testing/file_test/test_file.cpp

```diff
@@ -237,6 +237,27 @@ static auto ReplaceLspKeywordAt(std::string& content, size_t keyword_pos,
   return keyword_pos + json_with_header.size();
 }
 
+// Replaces `[[@0xAB]]` with the raw byte with value 0xAB.
+static auto ReplaceRawByteKeywordAt(std::string& content, size_t keyword_pos)
+    -> ErrorOr<size_t> {
+  llvm::StringRef content_at_keyword =
+      llvm::StringRef(content).substr(keyword_pos);
+  auto [keyword, rest] = content_at_keyword.split("]]");
+  if (keyword.empty()) {
```

Comment:
IIUC when the separator is not present, `split` returns an empty second result, so shouldn't this be something like `keyword == content_at_keyword`?